test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -147,12 +149,108 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
             }
         }
 
+        private static FileInfo CreatePackageFile(TestDirectory testDirectory)
+        {
+            var packageContext = new SimpleTestPackageContext();
+            var packageFile = new FileInfo(Path.Combine(testDirectory, "Package.nupkg"));
+
+            using (var readStream = packageContext.CreateAsStream())
+            using (var writeStream = packageFile.OpenWrite())
+            {
+                readStream.CopyTo(writeStream);
+            }
+
+            return packageFile;
+        }
+
+        private static SignatureContent CreateSignatureContent(FileInfo packageFile)
```

Comment:
You dont need this if you use `SignedArchiveTestUtility.CreateSignatureForPackageAsync`. See the comment below as well.

Reply:
It is critical that this test not call that helper method.  That helper method creates an author signed package.  The intent of this test is to verify basic signed CMS signature (where `SignatureType` is `Unknown`).  The signature generated by this test does not have any of the signed attributes which are required in an author signature.

I don't see value in making helper methods in `SignedArchiveTestUtility` for this one-time special need, though if there is in the future need to do something similar we should create them there.